src/quo2/components/settings/category/component_spec.cljs

```diff
@@ -0,0 +1,24 @@
+(ns quo2.components.settings.category.component-spec
+  (:require [quo2.components.settings.category.view :as category]
+            [test-helpers.component :as h]))
+
+(h/describe "Category list tests"
+  (h/test "Default render a Label on list component"
+    (h/render [category/category {:label "Gender"}])
+    (h/is-truthy (h/get-by-label-text :Gender)))
+
+  (h/test "It renders a list with items"
+    (h/render [category/category
+               {:settings-list-data [{:title               "Male"
+                                      :accessibility-label :settings-list-item
+                                      :left-icon           :browser-context
+                                      :chevron?            true
+                                      :border              true
+                                      :on-press            (fn [] (js/alert "Male pressed"))}
```

Comment:
component tests should not use actual functions, you need to use mock functions. Please take a look at some other tests. Alternatively if this prop is not part of the test then it should not be present in the data. Please only add the minimum props required for the specific test(s)